src/commands/run.rs

```diff
@@ -607,7 +607,21 @@ impl RunCommand {
         store: &mut Store<Host>,
         module: &RunTarget,
     ) -> Result<()> {
-        if self.run.common.wasi.common != Some(false) {
+        let mut cli = self.run.common.wasi.cli;
+
+        // Accept -Scommon as a deprecated alias for -Scli.
+        if let Some(common) = self.run.common.wasi.common {
+            if cli.is_some() {
+                bail!(
+                    "The -Scommon option should not be use with -Scli as it is a deprecated alias"
+                );
+            } else {
+                eprintln!("warning: The -Scommon flag has been renamed to -Scli");
```

Comment:
Could this be omitted for the time being? I figure it might be good to let the change percolate before warnings start showing up.